### ISSUE TITLE
Improve usability of slider selector by allowing to specify the slider position

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#932]
Currently, it is very very very difficult to precisely control slider position for small values (ie when range is 0 to 2000, it is very hard to set sliders to range 0-1, for example).

**Comment by pgundluru** [comment]
Currently the enhancement is applied to the below filters and while the slider function works the clear doesn't seem to clear the selection in the box.

Original filter (slice thickness) - Clear slider doesn't clear the selection in the box (the slider selected does clear)

Derived (quantitative analysis all filters) - Clear slider doesn't clear the selection in the box (the slider selected does clear)

Related (Age at diagnosis) - Clear slider doesn't clear the selection in the box (the slider selected does clear)

**Comment by G-White-ISB** [comment]
The new slider text boxes allow the user to input slider values, but the text box values would not necessarily sync with the slider range if the slider was updated by 'sliding' or pressing 'Clear slider'. I have a PR changing this behavior such that a change to the slider values will update the corresponding text boxes. 

Also note: user must press enter in one text box after values are entered in order for the boxes to update the slider. I'm hoping that is intuitive

**Comment by s-paquette** [comment]
@pgundluru Now ready for testing

**Comment by s-paquette** [comment]
@fedorov We could make a new ticket to improve the UI of it, or keep this one open and move it into the next release, as you prefer.

**Comment by fedorov** [comment]
I'd just keep this one open, to simplify matters.

**Comment by G-White-ISB** [comment]
" chances of anyone guessing that this functionality exists behind the gear icon are close to zero" - Yes but that would be true for other functionality behind the gear icons, ie 'Sort by alpha, num'.  Do you want to drop the gear icon in  the slider panels and have all slider controls statically visible?

**Comment by fedorov** [comment]
> Yes but that would be true for other functionality behind the gear icons

I do not agree with this. The key difference is that slider is visible without having to touch the gear icon. Other items you mentioned are not.

**Comment by fedorov** [comment]
But of course - all of this is subjective. It is not a priority to spend a lot of time on discussing or implementing this, all things considered.
